pc-app/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "ats-app",
+  "productName": "ATS-app",
+  "version": "1.0.0",
+  "description": "ATS Desktop Application",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "make": "electron-forge make",
+    "publish": "electron-forge publish",
+    "lint": "eslint src --max-warnings=0"
+  },
+  "keywords": [],
+  "author": {
+    "name": "Amin Jalilov",
+    "email": "aminkapkan444@gmail.com"
+  },
+  "license": "MIT",
```

Comment:
Doesn't match our [license](https://github.com/cs350-team4/ats/blob/master/LICENSE)

Reply:
Thanks for noticing